pkg/provider/azure_loadbalancer.go

```diff
@@ -2173,7 +2173,7 @@ func (az *Cloud) buildHealthProbeRulesForPort(serviceManifest *v1.Service, port
 				}
 			}
 		} else {
-			if port >= 0 || port <= 65535 {
+			if port >= 0 && port <= 65535 {
```

Comment:
could you fix the bug in master branch first? thanks.
https://github.com/kubernetes-sigs/cloud-provider-azure/blob/d9b43e25a13be5e01e8eda90584525e07ee8a6ae/pkg/provider/azure_loadbalancer.go#L2176

Reply:
ok, the pr to fix in master branch is https://github.com/kubernetes-sigs/cloud-provider-azure/pull/3339